metrics.go

```diff
@@ -223,19 +223,22 @@ func (r *RealtimeMetrics) Merge(other *RealtimeMetrics) {
 	}
 }
 
+// BucketsScanInfo holds scan information summary of all buckets
+type BucketsScanInfo struct {
+	OngoingBuckets int
+}
+
 // ScannerMetrics contains scanner information.
 type ScannerMetrics struct {
 	// Time these metrics were collected
 	CollectedAt time.Time `json:"collected"`
 
-	// Current scanner cycle
-	CurrentCycle uint64 `json:"current_cycle"`
+	CurrentCycle      uint64      `json:"current_cycle"`        // Deprecated Sep 2023
+	CurrentStarted    time.Time   `json:"current_started"`      // Deprecated Sep 2023
+	CyclesCompletedAt []time.Time `json:"cycle_complete_times"` // Deprecated Sep 2023
```

Comment:
Why is this deprecated?

Reply:
> Why is this deprecated?

I didn't see this; well, since a new bucket will be scanned when there is an available disk; there is no concept of scan cycle anymore; let's discuss this futher if you want